ssl/config.h

```diff
@@ -45,7 +45,7 @@
 #define CONFIG_SSL_HAS_PEM 1
 #undef CONFIG_SSL_USE_PKCS12
 #define CONFIG_SSL_EXPIRY_TIME 24
-#define CONFIG_X509_MAX_CA_CERTS 150
+#define CONFIG_X509_MAX_CA_CERTS 10
```

Comment:
Missed this the first time doing the review... `CONFIG_X509_MAX_CA_CERTS` is the max number of *root* certificates used for verification. It doesn't limit or set the maximum length of the certificate chain.